harness/src/main/java/org/jboss/pvt/harness/configuration/PVTConfiguration.java

```diff
@@ -8,10 +8,29 @@
  */
 public interface PVTConfiguration
 {
+    public static final String PRODUCT_VERSION_KEY = "PRODUCT_VERSION";
+    public static final String PRODUCT_VERSION_LAST_KEY = "PRODUCT_VERSION_LAST";
```

Comment:
If this PR is for PRODCI-29 then please isolate any configuration changes into a separate PR. Further, discussions in PRODCI-24 contain proposals for an actual configuration api as opposed to a set of Property keys.